l10n_it_declaration_of_intent/models/declaration.py

```diff
@@ -97,7 +96,12 @@ def create(self, vals_list):
             #       to create an in declaration
             # Declaration issued by company are "IN"
             if values.get("type", False) == "in":
-                year = fields.Date.to_date(values["date_start"]).strftime("%Y")
+                if isinstance(values["date_start"], dt.date):
+                    year = str(values["date_start"].year)
+                else:
+                    year = datetime.strptime(values["date_start"], "%Y-%m-%d").strftime(
```

Comment:
Sarebbe meglio usare `fields.Date.to_date` in modo da non dover _hardcodare_ il formato della data